### PR TITLE
add CRUD feature

### PR DESCRIPTION
**Pull request by Kemal-Wuzhi**
Add new CRUD related features.

### CODE REVIEW

**Comment by lochuhsin** [comment]
feat: add crud user feature ## Title

File: main.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    user["email"]=email
+    return user
+
+# PUT method 
+@app.put("/api/user/{user_id}")
+def put_user(name:str,email:str):
+    user["name"]=name
+    user["email"]=email
+    return user
+
```
Comment (lochuhsin) [inline main.py:28]: http:www.google.com/?user=1&name=John&birthday_month=12. # GET method

In general this should be 

http://localhost:8000/api/user/
send data using json.

**Review by lochuhsin — APPROVED** [review]

File: main.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+def post_user(user_data:User):
+    # check if it conforms User pydantic validation
+    return {"user_id": user_data.user_id, "name": user_data.name, "email": user_data.email}
+
+# @app.post("/api/user")
+# def post_user(user_id:User["user_id"], name:User["name"],email:User["email"]):
+#     user["user_id"]=user_id
+#     user["name"]=name
+#     user["email"]=email
+#     return user
```
Comment (lochuhsin) [inline main.py:53]: remove this

**Review by lochuhsin — APPROVED** [review]

**Comment by lochuhsin** [comment]
Basically, everything is decent in this level. Good Job

**Review by lochuhsin — APPROVED** [review]

File: main.py (hunk 2 above)
Comment (Kemal-Wuzhi) [reply main.py]: removed

**Pull request merged**